### ISSUE TITLE
Luminosity contrast ratio of focus indicator on 'Zoom in' button is 2.926:1 which is less than 3:1 in windows.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#35042]
### Describe the bug

**Test Environment:**
OS: Windows 11 Version 25H2 (OS Build 26200.8390)
Browsers: Edge (Version 148.0.3967.70), Chrome (Version 148.0.7778.168)
Tool: Accessibility insights for windows - color contrast tool
URL: https://storybooks.fluentui.dev/react/?path=/docs/charts_charts-areachart--docs

**User Impact:** 
Low vision users will be impacted as the low contrast focus indicator makes it difficult to visually identify the focused state on the “Zoom in” button. This can cause confusion and reduce usability, especially when navigating using keyboard.

**Actual Result:**
Luminosity contrast ratio of focus indicator on 'Zoom in' button is 2.926:1 which is less than 3:1 in windows.

**Refer Attachment:**
Luminosity contrast ratio of focus indicator on 'Zoom in' button is 2.926:1 which is less than 3:1 in windows.png

<img width="1346" height="961" alt="Image" src="https://github.com/user-attachments/assets/a278e3ed-ed5f-46e8-aff9-935e04aa7f59" />

**Expected Result:**
Luminosity contrast ratio of focus indicator on 'Zoom in' button should be greater than or equal to 3:1 in windows.

### Reproduction link

https://storybooks.fluentui.dev/react/?path=/docs/charts_charts-areachart--docs

### Reproduction steps

1. Open the URL: https://storybooks.fluentui.dev/react/?path=/docs/charts_charts-areachart--docs in edge/chrome browser.
2. Area Chart page will appear.
3. Press tab key to reach to the 'Zoom in' control present under 'Area Chart Basic' section.
4. Run 'Accessibility insights for windows - color contrast' tool.
5. Verify whether luminosity contrast ratio of focus indicator on 'Zoom in' button is 2.926:1 which is less than 3:1 in windows or not.

### System

```bash
npx storybook@latest 9.1.17
```

**Comment by Sidnioulz** [comment]
Thanks for the report @msftedad!

Could you please make sure that any new accessibility bug report is tested against Storybook 10.2 or newer? We have gone through a massive overhaul of color contrasts and fixed over 1000 reported contrast violations in our test suite.

In Storybook 10.4, I cannot reproduce this issue as the contrast of the focus outline is now sufficient. Thanks!

<img width="288" height="245" alt="Image" src="https://github.com/user-attachments/assets/15b9029d-d250-401c-b1dd-7c91552aea82" />

**Comment by msftedad** [comment]
Hi @Sidnioulz Thanks for the update. Could you please share the link of the environment where the issue has been fixed? We are still able to reproduce the issue on our end. Once we verify the fix, we'll close our bug accordingly.
